rclc/include/rclc/node.h

```diff
@@ -49,6 +49,33 @@ rclc_node_init_default(
   const char * namespace_,
   rclc_support_t * support);
 
+/**
+ *  Creates a RCL node with options.
+ *
+ *  * <hr>
+ * Attribute          | Adherence
+ * ------------------ | -------------
+ * Allocates Memory   | Yes (in this function and in RCL)
+ * Thread-Safe        | No
+ * Uses Atomics       | No
+ * Lock-Free          | Yes
+ *
+ * \param[in] name the name of the node
+ * \param[in] namespace the namespace of the node
+ * \param[in] support the rclc_support_t object
+ * \param[in] node_ops node options
+ * \return rcl_node_t if successful
+ * \return NULL if an error occurred
+ */
+rcl_ret_t
+rclc_node_init_with_options(
```

Comment:
Please add for every new function also a unit test in the corresponding unit test file.

Reply:
Working on it, @jamoralp is preparing a PR with these tests. Sorry for that Jan, we were in a hurry for these options and I forgot to implement the tests.

Reply:
Done. Please check #46 . Sorry for the inconvenience.